Source/WebCore/PAL/pal/PALSwift/UnsafeOverlays.swift

```diff
@@ -265,9 +269,45 @@ extension Curve25519.KeyAgreement.PrivateKey {
         if pubSpan.empty() {
             throw UnsafeErrors.emptySpan
         }
-        let pub =  try Curve25519.KeyAgreement.PublicKey(rawRepresentation: Data.temporaryDataFromSpan(spanNoCopy: pubSpan))
+        let pub = try Curve25519.KeyAgreement.PublicKey(
+            rawRepresentation: Data.temporaryDataFromSpan(spanNoCopy: pubSpan))
```

Comment:
Ditto.

Reply:
All the cosmetic changes in swift files are a result of `Tools/Script/swift/format <file>`. 
I am just letting the tool decide rather than me deciding what is right.